microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/jaxrs/JaxRsUtils.java

```diff
@@ -133,12 +134,16 @@ public static CodeLens createURLCodeLens(String baseURL, String rootPath, String
 	}
 
 	private static CodeLens createURLCodeLens(IMethod method, IJDTUtils utils) throws JavaModelException {
-		ISourceRange r = method.getNameRange();
-		if (r == null) {
+		IAnnotation[] annotations = method.getAnnotations();
+		if (annotations == null) {
 			return null;
 		}
+		ISourceRange r = annotations[annotations.length - 1].getSourceRange();
+
 		CodeLens lens = new CodeLens();
-		final Range range = utils.toRange(method.getOpenable(), r.getOffset(), r.getLength());
+		Range range = utils.toRange(method.getOpenable(), r.getOffset(), r.getLength());
+		range.setStart(new Position(range.getStart().getLine() + 1, range.getStart().getCharacter()));
+		range.setEnd(new Position(range.getEnd().getLine() + 1, range.getEnd().getCharacter()));

```

Comment:
This is done to get the line immediately below the last annotation right ? I would add a comment here explaining that.